packages/q/qt6gui/xmake.lua

```diff
@@ -8,6 +8,7 @@ package("qt6gui")
 
         if package:is_plat("linux") then 
             package:add("deps", "freetype", "fontconfig", "libxkbcommon")
+            add_extsources("apt::libgl1-mesa-dev", "pacman::mesa")
```

Comment:
don't do this, extsources is made to tell xmake that if a system package is installed it may be used instead of this one, which isn't the case here